Assets/Scripts/EnemySystem/ScriptableObjects/EnemyMovementStrategies/FollowTargetStrategy.cs

```diff
@@ -15,20 +15,23 @@ public class FollowTargetStrategy : EnemyMovementStrategy
 
         #endregion
 
-        public override void Move(Transform emuTransform, Transform playerTransform)
+        public override bool Move(Transform emuTransform, Transform playerTransform)
         {
-            if (Vector2.Distance(emuTransform.position, playerTransform.position) < threshold) return;
+            //This prevents the emu to be on top of the player
+            if (Vector2.Distance(emuTransform.position, playerTransform.position) < threshold) return false;
```

Comment:
what do you mean by on top? Doesn't the player pass through the emu because of the trigger box collider?

Reply:
remember the bug where melee emus were stuck on the top of the player?

when we worked on babe, we asked Will what was that line. you told me to add that line and said it was OK D:

Reply:
ooooooooooooooo yes i remember